tests/unit/dom/select-test.js

```diff
@@ -0,0 +1,146 @@
+import { module, test } from 'qunit';
+import { select, setupContext, teardownContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+import hasEmberVersion from '@ember/test-helpers/has-ember-version';
+import { isIE11 } from '../../helpers/browser-detect';
+
+let clickSteps = ['focus', 'focusin', 'input', 'change'];
+
+if (isIE11) {
+  clickSteps = ['focusin', 'input', 'change', 'focus'];
```

Comment:
did you test this?

Reply:
I haven't personally, but again this was on the other tests so felt that it was appropriate to add to this.

Reply:
we absolutely have to ensure that this is the right event order in the browsers. please do make sure that this is correct, otherwise the test helper will behave incorrectly.

Reply:
I validated against FF, Chrome, Safari, and IE11. See my findings in https://github.com/emberjs/ember-test-helpers/pull/728#pullrequestreview-414433140.